nodes/list_main/statistics.py

```diff
@@ -17,38 +17,126 @@
 # ##### END GPL LICENSE BLOCK #####
 
 import bpy
-from bpy.props import EnumProperty, IntProperty, FloatProperty, BoolProperty
+import blf
+import os
+
+from bpy.props import (BoolProperty,
+                       BoolVectorProperty,
+                       EnumProperty,
+                       FloatProperty,
+                       FloatVectorProperty,
+                       IntProperty,
+                       StringProperty)
 
 from sverchok.node_tree import SverchCustomTreeNode
-from sverchok.data_structure import updateNode, match_long_repeat
+from sverchok.data_structure import node_id, updateNode, match_long_repeat
 from sverchok.utils.modules.statistics_functions import *
-
-functions = {
-    "ALL_STATISTICS":     (0, 0),
-    "SUM":                (10, get_sum),
-    "SUM_OF_SQUARES":     (11, get_sum_of_squares),
-    "SUM_OF_INVERSIONS":  (12, get_sum_of_inversions),
-    "PRODUCT":            (13, get_product),
-    "AVERAGE":            (14, get_average),
-    "GEOMETRIC_MEAN":     (15, get_geometric_mean),
-    "HARMONIC_MEAN":      (16, get_harmonic_mean),
-    "STANDARD_DEVIATION": (17, get_standard_deviation),
-    "ROOT_MEAN_SQUARE":   (18, get_root_mean_square),
-    "SKEWNESS":           (19, get_skewness),
-    "KURTOSIS":           (20, get_kurtosis),
-    "MINIMUM":            (21, get_minimum),
-    "MAXIMUM":            (22, get_maximum),
-    "MEDIAN":             (23, get_median),
-    "PERCENTILE":         (24, get_percentile),
-    "HISTOGRAM":          (25, get_histogram)
```

Comment:
This actually breaks back compatibility and requires either MK_next version or adding new functions to the list end. Changes should not change existent properties.

Reply:
hm... so a MK2 is needed instead ?